packages/toolkit/config/webpack.config.js

```diff
@@ -49,13 +49,10 @@ const config = {
 	defaultTargets,
 };
 
-module.exports = {
+const baseConfig = {
```

Comment:
Let's only spin up two webpack configs/process if users pass a flag to avoid eating up resources for projects who don't need it.

Since this is very tied to the interactivity api I'd say we add something like: `--interactivity-api`

Reply:
Done. I added a `useBlockModules` setting for `package.json` and a `--block-modules` cli flag :)